nixos/modules/misc/documentation.nix

```
@@ -1,30 +1,54 @@
-{ config, lib, pkgs, extendModules, noUserModules, ... }:
+{ config, lib, options, pkgs, baseModules, extraModules, modules, modulesPath, utils, ... }:
```

Extremely minor nit, but since `config` and `options` are created in sequence in most modules (e.g. `config = {};` and then `options = {}`), and since I'd put `utils` and `modules` as the same "priority" as `config`, `options`, etc (mostly just because they're all lowercase, not snake case)...:

```suggestion
{ config, options, lib, pkgs, utils, modules, baseModules, extraModules, modulesPath, ... }:
```

---

I either sort alphabetically or don't care.